setup.py

```diff
@@ -46,6 +46,8 @@ def find_version(*file_paths):
     'cylc-flow>=8.0b0',
     'jupyterhub==1.3.*',
     'tornado==6.1.*',
+    'traitlets>=5.0.5',
+    'graphene>=2.1.8',
```

Comment:
What I understood was to have

```suggestion
    'traitlets',
    'graphene',
```

But the `graphene` after `graphene-tornado` I think?

Reply:
That's what I've done, isn't it?

Reply:
Maybe you didn't refresh your browser tab?

Reply:
(Oh, you probably reviewed just before I reverted the previous change then pushed the new one, sorry)